src/components/media-player/track-mixer-editor.yml

```diff
@@ -2,6 +2,9 @@ trackMixerEditor:
   noTrack:
     en: no track
     de: keine Spur
-  manageVolumePresets:
-    en: Manage volume presets
-    de: Lautstärke-Voreinstellungen bearbeiten
+  volumePresetInfo:
+    en: Sets of track volume combinations. Adjust the track volumes to set up the current preset.
+    de: Sets of track volume combinations, ja? :)
```

Comment:
Hier kannst du Sets von Lautstärke-Kombinationen definieren. Die einzelnen Spurlautstärken der momentan ausgewählten Kombination kannst im Spur-Mixer unten bearbeiten.

Reply:
Translated this in English too, thanks! 👍